easybuild/easyconfigs/i/iomkl/iomkl-2017.01-GCC-5.4.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2017.01'
+gcc_maj_min = '5.4'
+versionsuffix = '-GCC-%s' % gcc_maj_min
```

Comment:
Btw, should i drop it  even on hwloc-1.11.4-iccifort-2017.1.132-GCC-5.4.0-2.26.eb and similar?